spec/components/schemas/BroadcastMessages/BroadcastMessage.yaml

```
@@ -0,0 +1,101 @@
+type: object
+required:
+  - emails
+  - startSendingTime
+properties:
+  id:
+    description: The broadcast message's identifier string
+    readOnly: true
+    allOf:
+      - $ref: "#/components/schemas/ResourceId"
+  criteria:
+    $ref: "#/components/schemas/Condition"
+  title:
+    type: string
+    description: The title of the messages. (This title will appear in reports.)
+  messages:
+    type: array
+    description: The list of messages.
+    minItems: 1
+    items:
+      type: object
+      properties:
+        credentialId:
+          type: string
+          description: SMTP or Mailgun credential identifier string.
+        version:
+          type: string
+          description: The message title. (By default is empty, but if you make multiple versions, you can name or number them to distinguish them.)
+        from:
+          type: string
+          description: The sender address.
+          maxLength: 254
+        subject:
+          type: string
+          description: The message subject. The template placeholders are allowed.
+          maxLength: 998
+        text:
+          type: string
+          description: |
+            The message's text body.
+            Leave empty to use content from "html".
+            The template placeholders are allowed.
+        html:
+          type: string
+          description: |
+            The message's html body.
+            Leave empty to use content from "text".
+            The template placeholders are allowed.
+        language:
+          type: string
+          description: The message's ISO 639-1 two-letter language code.
+        unsubscribeLink:
```

```suggestion
        hasUnsubscribeLink:
```
The name is confusing, I expected url here. Maybe add prefix?

---

Will `includeUnsubscribeLink` work better?